test/utils/evaluate-testcase.js

```diff
@@ -69,6 +69,13 @@
         keys.forEach(function (key) {
             if (o.hasOwnProperty(key)) {
                 result[key] = sortedObject(o[key]);
+
+                // Nullify regex value for ES6 regex flags
+                if (key === 'value' && o.hasOwnProperty('regex')) {
+                    if ('uy'.indexOf(o.regex.flags) >= 0) {
```

Comment:
0. This will only work if there's a single flag or if the flags are exactly `uy`, in that order.
0. Why have this line at all? Just omit any regexp values when the node has a `regex` field.